modules/ptp-dual-nics.adoc

```diff
@@ -29,3 +29,11 @@ Dual NIC boundary clock::
 For 5G telco networks that deliver mid-band spectrum coverage, each virtual distributed unit (vDU) requires connections to 6 radio units (RUs). To make these connections, each vDU host requires 2 NICs configured as boundary clocks.
 +
 Dual NIC hardware allows you to connect each NIC to the same upstream leader clock with separate `ptp4l` instances for each NIC feeding the downstream clocks.
+
+Highly available system clock with dual NIC boundary clocks::
+You can configure Intel E810-XXVDA4 Salem channel dual NIC hardware as dual PTP boundary clocks with highly available system clock where each NIC is connected to the same upstream leader clock.
```

Comment:
"dual PTP boundary clocks with highly available system clock"

I feel like the plural agreement is off or an article is missing here, but I am not sure what direction to go in

Reply:
It's a HA system, so both boundary clocks are connected. The primary boundary clock feeds the system clock. If the primary clock goes offline, the secondary clock takes over feeding the system clock.

Reply:
(I reworded this slightly, hopefully this is clearer now)

Reply:
LGTM, thanks! I can see how this topic is a bit hard to talk about :sweat_smile: